tests/qiskit_qir/conftest.py

```
@@ -0,0 +1,166 @@
+# limitations under the License.
+
+"""
+Pytest fixtures for Qiskit QIR tests.
+
+"""
+
+import pytest
+from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister
+
+
+@pytest.fixture()
```

Ruff (0.15.2)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)